galley/pkg/config/meta/schema/codegen/tools/staticinit.main.go

```diff
@@ -14,15 +14,15 @@
 
 // +build ignore
 
-package main
+package tools
 
 import (
 	"fmt"
 	"io/ioutil"
 	"os"
 
-	"istio.io/istio/galley/pkg/config/schema"
-	"istio.io/istio/galley/pkg/config/schema/codegen"
+	"istio.io/istio/galley/pkg/config/meta/schema"
+	codegen2 "istio.io/istio/galley/pkg/config/meta/schema/codegen"
```

Comment:
Is there a reason we need the import renames here and elsewhere? Is this leftover intermediate state?

Reply:
No, this is introduced by the auto-refactoring that the IDE did. I just fixed them.